stackdriver-core/lib/stackdriver/core/async_actor.rb

```diff
@@ -258,9 +258,8 @@ def self.unregister_for_cleanup actor
       #
       def self.run_cleanup
         @exit_lock.synchronize do
-          if @cleanup_list
-            @cleanup_list.shift.async_stop! until @cleanup_list.empty?
-          end
+          return unless @cleanup_list
```

Comment:
@dazuma plz verify this change. Rubocop wants to nest the conditionals as `until @cleanup_list.empty? if @cleanup_list`, but then complains about nested conditionals...

Reply:
That nested conditionals case should probably be considered a rubocop bug. I do like your solution.